src/sql/parts/profiler/contrib/profilerActions.ts

```diff
@@ -88,7 +89,9 @@ export class ProfilerStart extends Action {
 
 export class ProfilerPause extends Action {
 	public static ID = 'profiler.pause';
-	public static LABEL = nls.localize('pause', "Pause");
+	public static LABEL = nls.localize('pause', "Capture");
```

Comment:
the first parameter to `localize` needs to be globally unique across the project.  I think `pause` is possible to hit collisions at some point.  Perhaps something like `nls.localize('profiler.capture', "Capture");` or similar would be more specific.